src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/SyntaxKindEx.cs

```diff
@@ -12,12 +12,18 @@ namespace Microsoft.CodeAnalysis.CSharp.Formatting
     /// </summary>
     internal static class SyntaxKindEx
     {
-        // The code style layer does not currently need access to any syntax defined in newer versions of Roslyn. This
-        // type is included as an example should this change in future updates.
+        public const SyntaxKind DataKeyword = (SyntaxKind)8438;
+
+#if !CODE_STYLE
+        private const uint DotDotTokenValueAssertion = -(DataKeyword - SyntaxKind.DataKeyword);
+#endif
```

Comment:
what's this?

Reply:
This is explained below. I forgot to rename the constant though. Will fix